azurerm/data_source_monitor_action_group_test.go

```diff
@@ -170,7 +220,41 @@ resource "azurerm_monitor_action_group" "test" {
   webhook_receiver {
     name        = "callmybackupapi"
     service_uri = "https://backup.example.com/warning"
+	}
+
+	automation_runbook_receiver {
+		name = "action_name_1"
+		automation_account_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001"
```

Comment:
could we provision [an automation account](https://www.terraform.io/docs/providers/azurerm/r/automation_account.html) dynamically too and reference it's ID here?